components/ntp_background_images/browser/view_counter_service.cc

```diff
@@ -504,6 +517,8 @@ bool ViewCounterService::CanShowSponsoredImages() const {
     if (base::Time::Now() < installation_date + *images_data->grace_period) {
       // We don't show sponsored images if the user has installed Brave within
       // the grace period.
+      LOG(INFO) << "Sponsored images not shown: Grace period after "
+                   "installation is still active.";
```

Comment:
Added to help QA test a fresh install, otherwise it would look to the user like no ads are being served.